migration/identity-migration/src/main/java/io/camunda/migration/identity/TenantMappingRuleMigrationHandler.java

```diff
@@ -57,6 +57,7 @@ protected MigrationStatusUpdateRequest processTask(final TenantMappingRule tenan
           new MappingDTO(
               tenantMappingRule.getClaimName(),
               tenantMappingRule.getClaimValue(),
+              tenantMappingRule.getName(),
```

Comment:
❓ Do we have any special considerations around what is an `id` for example can it contain spaces? do we need to, as an example, convert this to all lower case and replace spaces with `-`?